frontend/src/views/activity/Category.vue

```diff
@@ -53,13 +53,13 @@
 
 <script>
 import ContentCard from '@/components/layout/ContentCard.vue'
-import ContentNode from '@/components/activity/ContentNode.vue'
+import RootNode from '@/components/activity/ContentNode.vue'
```

Comment:
Ich glaube du hast vergessen, den Import anzupassen. Aktuell wird der neue `RootNode` nie verwendet.
```suggestion
import RootNode from '@/components/activity/RootNode.vue'
```